### PR TITLE
Expiration check (EXPOSUREAPP-8368) 

### PR DESCRIPTION
**Pull request by chiljamgossow**
Checks expiration and returns respective state

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/signature/core/DccStateChecker.kt [hunk 1]
```diff
@@ -22,10 +25,29 @@ class DccStateChecker @Inject constructor(
     suspend fun checkState(
         dccData: DccData<*>
     ): Flow<CwaCovidCertificate.State> = flow {
-        // TODO
-        val state = CwaCovidCertificate.State.Valid(
-            expiresAt = Instant.now().plus(Duration.standardDays(21))
-        )
+        // TODO signature check
```
Comment (SamuraiKek) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/signature/core/DccStateChecker.kt:28]: Will this be done in a sepparate PR?

Comment (d4rken) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/signature/core/DccStateChecker.kt:28]: Part of https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8361

**Review by d4rken — APPROVED** [review]
lgtm, good test coverage :heart:

**Review by SamuraiKek — APPROVED** [review]
Lgtm 🚀

**Pull request merged**